airflow/providers/databricks/provider.yaml

```
@@ -59,7 +59,7 @@ versions:
 
 dependencies:
   - apache-airflow>=2.6.0
-  - apache-airflow-providers-common-sql>=1.8.1
+  - apache-airflow-providers-common-sql>=1.9.1
```

```suggestion
  - apache-airflow-providers-common-sql>=1.9.0
```

UPDATE: nope, it's good.

---

Here and in other places

---

Ah no.... scratch that... 1.9.1 is fine (looked at all version of provider.yaml of common.sql). Can you, however add 1.9.1 in the `provider.yaml` as it will not pass the tests. When two providers depends on each other with min-version, the min-version has to be added to provider.yaml of the provider that is depended on.